### PR TITLE
Resolves #2340

### LINKED ISSUE #2340: ExtranetLockoutMode
I think property "ExtranetLockoutMode" is missing from this page? Should be available with an updated ADFS 2016 or ADFS 2019. 

Check with **help Set-AdfsProperties**

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c4c16fe4-05b8-e965-8e28-59decee1edc1
* Version Independent ID: 575c608a-b919-a242-a404-526c95226a39
* Content: [Set-AdfsProperties (ADFS)](https://docs.microsoft.com/en-us/powershell/module/adfs/set-adfsproperties?view=windowsserver2019-ps)
* Content Source: [docset/winserver2019-ps/adfs/Set-AdfsProperties.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2019-ps/adfs/Set-AdfsProperties.md)
* Product: **w10**
* Technology: ****
* GitHub Login: @Kateyanne
* Microsoft Alias: **v-kaunu**

### PR DESCRIPTION
**Pull request by velkovb**
Add missing parameter
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2340

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — CHANGES_REQUESTED** [review]
Please also add the parameter to the parameters section

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

File: docset/winserver2016-ps/adfs/Set-AdfsProperties.md [hunk 1]
```diff
@@ -580,6 +580,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ExtranetLockoutMode:
+Specifies Extranet Smart Lockout mode
```
Comment (dariomws) [inline docset/winserver2016-ps/adfs/Set-AdfsProperties.md:584]:
```suggestion
Specifies Extranet Smart Lockout mode.
```

File: docset/winserver2019-ps/adfs/Set-AdfsProperties.md [hunk 2]
```diff
@@ -580,6 +580,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ExtranetLockoutMode:
+Specifies Extranet Smart Lockout mode
```
Comment (dariomws) [inline docset/winserver2019-ps/adfs/Set-AdfsProperties.md:584]:
```suggestion
Specifies Extranet Smart Lockout mode.
```

File: docset/winserver2022-ps/adfs/Set-AdfsProperties.md [hunk 3]
```diff
@@ -580,6 +580,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ExtranetLockoutMode:
+Specifies Extranet Smart Lockout mode
```
Comment (dariomws) [inline docset/winserver2022-ps/adfs/Set-AdfsProperties.md:584]:
```suggestion
Specifies Extranet Smart Lockout mode.
```

**Review by dariomws — APPROVED** [review]

**Pull request merged**